src/runtime/audience-action-flow.js

```diff
@@ -163,6 +164,8 @@ export class AudienceActionFlow {
     } else {
       this.showFailedOptedInToast_();
     }
+    const now = Date.now().toString();
+    this.deps_.storage().set(Constants.READ_TIME, now, true);
```

Comment:
I think we should actually be fine to put this in session storage. The max staleness that we are going to start with will just be 10s so its ok for this property to be ephemeral

Reply:
Done.